samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -79,14 +94,35 @@ export class FormComponent implements OnInit {
     this.submittingForm = true;
 
     try {
+      const selectIdCallback = step?.callbacks.find(
```

Comment:
should we not use the getCallbacksOfType here also?

Reply:
done